native/cocos/application/BaseGame.cpp

```diff
@@ -46,12 +48,23 @@ int BaseGame::init() {
                                                       cc::ISystemWindow::CC_WINDOW_INPUT_FOCUS
                                                 : _windowInfo.flags;
     std::call_once(_windowCreateFlag, [&]() {
-        if (_windowInfo.x == -1 || _windowInfo.y == -1) {
-            createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        } else {
-            createWindow(_windowInfo.title.c_str(),
-                         _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
-        }
+        //if (_windowInfo.x == -1 || _windowInfo.y == -1) {
+        //    createWindow(_windowInfo.title.c_str(), _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //} else {
+        //    createWindow(_windowInfo.title.c_str(),
+        //                 _windowInfo.x, _windowInfo.y, _windowInfo.width, _windowInfo.height, _windowInfo.flags);
+        //}
+
+        ISystemWindowInfo info;
+        info.title = _windowInfo.title;
+        info.x = _windowInfo.x == -1 ? 50 : _windowInfo.x;
```

Comment:
50: a magic number, why 50?

Reply:
So it fixed an existing issue?

Reply:
I'm not sure if it is an issue.

Reply:
> If 0 the title bar of window is outside the screen

@qiuguohua please check it. I think it is an issue.

Reply:
I added a comment for now